src/disco/css/Addon.scss

```diff
@@ -6,7 +6,7 @@ $addon-padding: 20px;
 .addon {
   align-items: center;
   background: #fff;
-  box-shadow: 0 1px 2px 0 rgba(0,0,0,0.1);
+  box-shadow: 0 1px 2px 0 rgba(0,0,0,0.2);
```

Comment:
nitpick, do we really don't care about whitespaces after the comma here? `rgba(0, 0, 0, 0.2)` reads a lot easier imho.